src/io/ImportData.cc

```diff
@@ -0,0 +1,58 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2021 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file ImportData.cc
+//---------------------------------------------------------------------------//
+#include "ImportData.hh"
+
+#include <algorithm>
+
+namespace celeritas
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Find an \c ImportElement object from \c ImportData using \c element_id .
+
+ * This simplifies the process of fetching full elemental information from a
+ * given material, as \c ImportMatElemComponent only stores the \c element_id .
+ */
+const ImportElement
+get_import_element(const ImportData& data, unsigned int element_id)
+{
+    CELER_EXPECT(element_id <= data.elements.size());
+
+    auto iter = std::find_if(data.elements.begin(),
+                             data.elements.end(),
+                             [element_id](const ImportElement& element) {
+                                 return element.element_id == element_id;
+                             });
```

Comment:
Shouldn't this just be `auto iter = data.elements.begin() + element_id`? The "id" of an element should always be its position in the list, if we're being consistent and operating on import data in the context of an `ImportFile`.